staubli_rx160_support/tests/roslaunch_test.xml

```diff
@@ -6,4 +7,12 @@
   <group ns="test_rx160__">
     <include file="$(find staubli_rx160_support)/launch/test_rx160.launch"/>
   </group>
+
+  <group ns="load_rx160l__">
```

Comment:
I don't think there is any documentation for that. IIRC @shaun-edwards started adding `roslaunch` tests and he came up with the namespace idea.